docs/windows.md

```diff
@@ -59,7 +59,54 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (Containerd based runtimes)
+
+First install Antrea (v0.13.0+ is required for Containerd).
+
+```bash
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea.yml
+```
+
+Then, you can run the following commands. This will install Antrea as a Windows service. Please ensure [nssm](https://nssm.cc/)
+is on your machine, which is a handy tool to manage service on Windows. NOTE: `<KubernetesVersion>`, `<KubeconfigPath>`
+and `<KubeletKubeconfigPath>` should be filled by you. E.g.
+```powershell
```

Comment:
you need an empty line here, or it won't pass CI (check for correct website formatting)

Reply:
Thanks for reminding. Updated.